src/rpc/server.cpp

```diff
@@ -162,7 +162,7 @@ UniValue stop(const JSONRPCRequest& jsonRequest)
     if (jsonRequest.fHelp || jsonRequest.params.size() > 1)
         throw std::runtime_error(
             RPCHelpMan{"stop",
-                "\nStop Bitcoin server.",
+                "\nStop the " PACKAGE_NAME " daemon.",
```

Comment:
The gui is not a daemon, so this should be kept more general

Reply:
I don't think changing this from server to daemon is an improvement to be honest.
(definitely not from a documentation perspective, "server" is something everyone understands while "daemon" is very much a UNIX word)

Reply:
How about "Requests a graceful shutdown"?